libs/payments/ui/src/lib/client/components/SignInForm/en.ftl

```diff
@@ -0,0 +1,7 @@
+continue = Continue
+checkout-enter-your-email = Enter your email
```

Comment:
I know these aren't technically landing in front of localizers yet, but I'd recommend namespacing these.

Reply:
Thanks for the feedback, @bcolsson! I've updated the ids as follows: 
```
signin-form-continue-button = Continue
signin-form-email-input = Enter your email
```